frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -311,6 +315,37 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
     );
   };
 
+  const actionText =
+    (detailsItem &&
+      _.get(detailsItem.obj, [
+        'status',
+        'channels',
+        '0',
+        'currentCSVDesc',
+        'annotations',
+        'marketplace.openshift.io/action-text',
+      ])) ||
```

Comment:
This should be read with the other annotations

https://github.com/spadgett/console/blob/bd4bf53545273a950023660b8482fc3ab60a6914/frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-page.tsx#L42-L50